src/test/resources/features/account/import_account.feature

```diff
@@ -0,0 +1,21 @@
+@Account
+Feature: Import Accounts
+
+  @deleteAccountImports
+  Scenario: Import multiples accounts using csv files
```

Comment:
I think tittle could be improved like "Import multiple Accounts using .csv files"